### PR TITLE
post: security scanning

### PR DESCRIPTION
**Pull request by mssola**
Added a blog post about the newly merged security scanning support. I've
also fixed a typo from the documentation of this feature.

Signed-off-by: Miquel Sabaté Solà <msabate@suse.com>

### CODE REVIEW

File: _posts/2017-07-19-security-scanning.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+title: Security scanning
+author: Miquel Sabaté Solà
+layout: blogpost
+---
+
+One of the perks of working at SUSE are [hackweeks](https://en.opensuse.org/Portal:Hackweek). A hackweek is a period
+of 5 days in which every employee is free to work on whatever they want. Portus
+was born this way, as Flavio explained on his [blog post](http://flavio.castelli.name/2015/04/23/introducing-portus-a-user-interface-for-docker-registry/), and we are happy
```
Comment (monstermunchkin) [inline _posts/2017-07-19-security-scanning.md]: "*in* his blog post"

File: _posts/2017-07-19-security-scanning.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+to say that this feature announced today was also started this way.
```
Comment (monstermunchkin) [inline _posts/2017-07-19-security-scanning.md]: this -> the

File: _posts/2017-07-19-security-scanning.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+with [zypper-docker](https://github.com/SUSE/zypper-docker), a tool that allows you to patch and monitor Docker
+images.
+
+Moreover, as other projects like [Docker hub](https://hub.docker.com/) and [Quay](https://quay.io/) have proven, it's
+pretty useful that the same web application that allows you to manage your
+Docker images, also fetches security information about them. Because of all
+this, we thought that it would be a good idea to create a new layer in Portus,
+in which we could integrate security tools, and finally report back about the
+security status of the images on your on-premise Docker registry. After toying
+with the idea, me and [Vítor Avelino](https://github.com/vitoravelino) worked on this and we have just
```
Comment (monstermunchkin) [inline _posts/2017-07-19-security-scanning.md]: "Vitor and I"

File: _posts/2017-07-19-security-scanning.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+- [zypper-docker](https://github.com/SUSE/zypper-docker): zypper-docker is a command line tool that provides a quick
+  way to patch and update Docker Images based on either SUSE Linux Enterprise or
+  openSUSE. In a hackweek we started to work on a `serve` command, so it can be
+  used as a server (see [this branch](https://github.com/SUSE/zypper-docker/tree/cli-separation)). However, we never managed to fully
+  stabilize this branch, so support for this is considered experimental.
+- The `dummy` backend: this backend is only meant to be used for development
+  purposes. That is, it allows developers to have some fixtures that they can
+  use.
+
+If you have enabled this feature, then in a repository you should be able to see
```
Comment (monstermunchkin) [inline _posts/2017-07-19-security-scanning.md]: "[...], you should be able to see something like this in a repository:"

File: _posts/2017-07-19-security-scanning.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+something like this:
+
+![Repository page with vulnerabilities highlighted](/build/images/docs/security.png)
+
+Clicking these links will take you the the tag's page. For example:
```
Comment (monstermunchkin) [inline _posts/2017-07-19-security-scanning.md]: "take you *to* the"

File: _posts/2017-07-19-security-scanning.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+![Tag page with Clair vulnerabilities](/build/images/docs/tag-vulnerabilities.png)
+
+## Availability
+
+This feature will be available as of Portus 2.3, but you can already try it out
+on the master branch. You can read more about this feature on
```
Comment (monstermunchkin) [inline _posts/2017-07-19-security-scanning.md]: "*in* the documentation."

File: _posts/2017-07-19-security-scanning.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+this, we thought that it would be a good idea to create a new layer in Portus,
+in which we could integrate security tools, and finally report back about the
+security status of the images on your on-premise Docker registry. After toying
+with the idea, [Vítor Avelino](https://github.com/vitoravelino) and I worked on this and we have just
+recently [merged it all into master](https://github.com/SUSE/Portus/pull/1289).
+
+## How to use it
+
+First of all, this feature is disabled by default, and the configuration part
+for it looks like this:
```
Comment (vitoravelino) [inline _posts/2017-07-19-security-scanning.md]: Maybe it's worth mentioning that it's in `config/config.yml`?

Comment (mssola) [inline _posts/2017-07-19-security-scanning.md]: Done

**Pull request merged**